src/notifications/ns_settings.py

```diff
@@ -43,21 +43,26 @@ def __init__(self, client):
     def loadSettings(self):
         self.enabled = Settings.get('notifications/enabled', True, type=bool)
         self.popup_lifetime = Settings.get('notifications/popup_lifetime', 5, type=int)
+        self.popup_position = Settings.get('notifications/popup_position', "bottom right", type=str)
 
         self.nsEnabled.setChecked(self.enabled)
         self.nsPopLifetime.setValue(self.popup_lifetime)
+        comboboxIndex = [i for i in range(4) if self.nsPositionComboBox.itemText(i) == self.popup_position]
+        self.nsPositionComboBox.setCurrentIndex(comboboxIndex[0] if comboboxIndex else -1)
```

Comment:
I think the key mistake here is a failure to separate combo box labels (a view thing) from the selected position (a model thing). Read up on "Movel/view pattern" if you want to get really detailed, I guess.

Don't store the labels, store some canonical representation. I'd suggest an enum to represent positions, and a map to map enum values to labels, like this:

```
class NotificationPosition(Enum):
    TOP_LEFT = 1
    BOTTOM_LEFT = 2
    BOTTOM_RIGHT = 3
    TOP_RIGHT = 4

    def getLabel(self):
        if self == NotificationPosition.TOP_LEFT:
            return "top left"
        elif self == NotificationPosition.BOTTOM_LEFT:
            return "bottom left"
        elif self == NotificationPosition.TOP_RIGHT:
            return "top right"
        elif self == NotificationPosition.TOP_RIGHT:
            return "top right"
```

For one thing, this makes localisation actually _possible_ without breaking the stored values (you'd just modify this to call into a localisation function. Your way, if they change language (or we relabel a button) the stored setting becomes invalid). It also means you don't need to do this weird list comprehension thing. You can now, given a `NotificationPosition`, just call `getLabel()` on it to get the needed label. You can trivially make a list of labels as a list comprehension, and you avoid storing localised strings in the settings file.